_posts/2019-05-15-the-quest-for-a-better-search.md

```diff
@@ -12,25 +12,25 @@ tags:
 category: soa-search
 ---
 
-As a developer if there is one thing I should blame Google for is the distortion they inflicted to the search experience for users. When customers requirements list things like “Google-like search capabilities” my comment is generally along the lines of:
+As a developer if there is one thing I should blame Google for it's the distortion they inflicted on the search experience for users. When customer requirements list things like “Google-like search capabilities” my comment is generally along the lines of:
 
 > Give me the Google budget, and you’ll have the Google search.
 
-Last week I was on a call with my colleague [Tomek](https://twitter.com/Masternak) to discuss search capabilities in a distributed system. The very first distinction that we made is:
+Last week I was on a call with my colleague [Tomek](https://twitter.com/Masternak) to discuss search capabilities in a distributed system. The very first distinction that we made was:
 
 > You are Google, you are not Google.
```

Comment:
Did you mean to say:

> You are not Google, you are not Google

Reply:
no, I meant to say that the first distinction is are you in the search business or not? If you are in the search business you're like Google, otherwise no.

Any better way way to say that?